test-packages/custom-fastboot-testing-app/tests/fastboot/home-page-test.js

```diff
@@ -0,0 +1,11 @@
+import { module, test } from 'qunit';
+import { setup, visit } from 'ember-cli-fastboot-testing/test-support';
+
+module('FastBoot | home-page', function(hooks) {
+  setup(hooks);
+
+  test('it renders a page...', async function(assert) {
```

Comment:
Let's add a better name to the test here.

Reply:
Done.